.github/workflows/deploy.yaml

```diff
@@ -13,6 +13,8 @@ jobs:
     environment: production
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
We now do a deep checkout so that we get the real number of commits in the branch. Will also need this for the subtree push in an upcoming PR